xbmc/guilib/guiinfo/GUIInfoLabels.h

```diff
@@ -194,6 +194,8 @@
 #define MUSICPLAYER_CONTRIBUTOR_AND_ROLE 240
 #define MUSICPLAYER_DBID            241
 #define MUSICPLAYER_PROPERTY        242
+#define MUSICPLAYER_DISC_TITLE      243
+#define MUSICPLAYER_IS_BOXSET       244
```

Comment:
Yeah, saw the comments over on upstream regarding that.  I'm not sure that MUSICPLAYER_IS_BOXSET is actually needed.  Certainly, I don't think it's set anywhere currently.  I did that a while ago when initially getting to grips with this.  However, I guess it might be useful for a skin to know if it's dealing with an ordinary album or a boxset (maybe different art, show all discart for set etc etc) so perhaps I should leave it and set appropriately ??

Reply:
Good question. It is a good principle that everthing a skin can know about an item in a "songs" node it can also know about the playing song. However LISTITEM_IS_BOXSET really applies to albums not songs, and it is hard to see that knowing that song in a list or the currently playing song is from a boxed set would be useful. 

I'm leaning towards removing it, but not aggresively

Reply:
Well as it's not used at the moment, it would seem reasonable to drop it, at least for now.  Skins can determine (in musicvis) if a track is from a boxset with !String.IsEmpty($INFO[MusicPlayer.DiscTitle])  which will only contain a value if bBoxedSet is true.

Reply:
> which will only contain a value if bBoxedSet is true.

I think that if the disc a song is on has a title then the player (and item list) should be able to show it regardless of being a boxed set or not

Reply:
It does.  If there is an actual title, rather than an auto-generated 'disc x' title, then that is set in MusicPlayer.DiscTitle and ListItem.DiscTitle.  That includes dummy 'disc x' titles generated when 'boxset' is set as a value in Musicbrainz_releasetype.  Basically, any disctitle that is stored in the db will appear in those two infolabels.  If the title was generated as part of the navigation (ie no actual disc title, just multiple discs) then it won't show in the infolabels.

I have tagged a couple of multi-disc albums with disc titles on the first and last discs but no titles on the intervening ones and in all cases the correct titles are displayed when present, even though these albums are NOT in the boxset node.

Reply:
Removed the unused MUSIC_PLAYER_IS_BOXSET and ListItem.IsBoxSet . Can always be added back later if they are needed/useful for skinners.